packages/modular-scripts/src/utils/getPackageDependencies.ts

```diff
@@ -46,19 +47,18 @@ export async function getPackageDependencies(
   const targetLocation = await getLocation(target);
   const workspaceInfo = getWorkspaceInfo();
 
-  const rootPackageJsonDependencies =
-    (
-      fs.readJSONSync(
-        path.join(getModularRoot(), 'package.json'),
-      ) as CoreProperties
-    ).dependencies || {};
+  const rootManifest = fs.readJSONSync(
+    path.join(getModularRoot(), 'package.json'),
+  ) as CoreProperties;
 
-  const targetPackageJsonDependencies =
-    (
-      fs.readJSONSync(
-        path.join(targetLocation, 'package.json'),
-      ) as CoreProperties
-    ).dependencies || {};
+  const targetManifest = fs.readJSONSync(
+    path.join(targetLocation, 'package.json'),
+  ) as CoreProperties;
+
+  const rootPackageJsonDependencies = rootManifest.dependencies || {};
+  const targetPackageJsonDependencies = targetManifest.dependencies || {};
+  const rootPackageJsonDevDependencies = rootManifest.devDependencies || {};
+  const targetPackageJsonDevDependencies = targetManifest.devDependencies || {};
```

Comment:
nit: as an option for making this a bit less verbose:

```javascript
const deps = Object.assign(Object.create(null).
  targetManifest.devDependencies,
  rootManifest.devDependencies, 
  targetManifest.dependencies, 
  rootManifest.dependencies, 
);

// ...
const devVersion = deps[depName];
if (depVersion === undefined) {
  logger.error(...);
}

manifest[depName]
```

Reply:
Done with 

```
const deps = {
    ...rootManifest.dependencies,
    ...targetManifest.dependencies,
    ...rootManifest.devDependencies,
    ...targetManifest.devDependencies,
  };
```